packages/cms-lib/lib/config.js

```diff
@@ -16,11 +16,33 @@ const {
 let _config;
 let _configPath;
 
+const getOrderedConfig = unorderedConfig => {
+  const {
+    defaultPortal,
+    defaultMode,
+    httpTimeout,
+    allowsUsageTracking,
+    portals,
+    ...rest
+  } = unorderedConfig;
+
+  return {
+    defaultPortal,
+    defaultMode,
+    httpTimeout,
+    allowsUsageTracking,
+    portals,
+    ...rest,
+  };
+};
```

Comment:
I was gonna bring up that key order isn't guaranteed, but it seems that may be more old knowledge I carry that is no longer valid.
https://www.stefanjudis.com/today-i-learned/property-order-is-predictable-in-javascript-objects-since-es2015/